Podcast/Info.plist

```diff
@@ -57,6 +57,10 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>audio</string>
```

Comment:
nice!

Reply:
This doesn't complete it haha. I'm missing one part in player, registering it. But we will need this.

Reply:
but one of two parts done!